jina/serve/executors/__init__.py

```diff
@@ -526,6 +709,7 @@ def serve(
             uses_with=uses_with,
             uses_metas=uses_metas,
             uses_requests=uses_requests,
+            reload=reload,
         )
```

Comment:
are u sure all these `kwargs` are passed to the Executor itself?

Reply:
I tried out the reload locally
Putting the executor definition and executor.serve() in the same file is not supported so I added a warning each time we attempt to reload the main module. In other cases it works

Reply:
kwargs are passed to the Flow. That way, the user of serve will have full power over everything (gateway + executor).
I confirm that kwargs are passed correctly to Flow and the Flow in return, passes them to gateway and executor (beisdes the ones explicitly passed to the executors)